tests/test_serialization.py

```diff
@@ -356,7 +356,7 @@ def test_boolean_field_serialization(self, user):
         assert field.serialize("none", user) is None
 
     def test_function_with_uncallable_param(self):
-        with pytest.raises(ValueError):
+        with pytest.raises(TypeError):
             fields.Function("uncallable")
```

Comment:
Isn't this test a duplicate of ```test_function_field_passed_uncallable_object`?

Reply:
Yeah looks like it, I'll go ahead and remove this one